### ISSUE TITLE
Documentation - monitor behaviour request. 

### ISSUE THREAD

**Issue by xpenguinclub** [conformal/spectrwm#239]
Hello, sorry for using a git issue for this but honestly, not sure how else to get an answer.

I have three screens and the behaviour i require to to firmly 'pin' specific workspace numbers on specific screens. 

left screen 1 2 3
middle screen 4 5 6 7
right screen 8 9 0 

with a relatively default setup the workspaces (not sure if thats how they are referred to in Spectwm, sorry)  move around between desktop depending on focus. 

The documentation briefly mentions 'clamp' variable and imply that it can clamp a workspace to a specific screen but does not expand on how to make that work.

can someone please assist or point me at a tutorial that makes sense please? - regardless of outcome of this, I really like the project and find its methodology fascinating. 

again, so sorry for using a git-issue for this, cant really see a better place to ask.

HexDSL

**Comment by xpenguinclub** [comment]
Okay, it would appear that i simply have to specify 

workspace_clamp = 1 
warp_pointer = 1 

bit now i cant specifiy what workspaces are on what monitors!  - plz help! :D

**Comment by LordReg** [comment]
workspace_clamp just prevents visible workspaces from swapping between regions when you switch workspaces.  warp_pointer moves the pointer when window focus changes.

By default, a separate region is created for each monitor in the order they appear in `xrandr --listactivemonitors`.  Each region has an initial active workspace, based on that order.
I'm not sure I completely understand what you are trying to do.   Do you want each region to have an independent set of workspaces or is setting the initial workspace on each region enough?

**Comment by hexdsl** [comment]
yeah, what i want is to have each workspace be fixed to a specific monitor.

so when i switch to (for example) workspace 8, it always appears on my right hand side monitor and (for example) 4 would always be on middle monitor. 

this is so that i can always have my web browser on the smae physical screen  and always have IRC on the right.

**Comment by LordReg** [comment]
so you basically want to have the ability to have `ws_` binds switch workspaces on specific regions (monitors) without needing to actually focus on the region?
for example, if ws 1-3 were 'pinned' to region 1 and you were focused on region 2, when you switch between ws 1, ws 2, ws3, the switching would occur on region 1 instead of region 2?

Any thoughts on how this would be controlled? I'm thinking a 'pin_ws_to_region' toggle action and some config option to allow a user to automatically pin workspaces at start.

**Comment by marcopeereboom** [comment]
Meh on my end but I don't mind something like this going in provided it does not mess with the defaults.

**Comment by jerome-diver** [comment]
@LordReg maybe by define it through layout config variable.
if you add the region inside the layout definition (at 1 by default), then it should be possible to manage this because at jump time (when yu change ws), it should be tyhen able to consider on wich region the layout is define and stack/manage the workspaces to be in front for each region only from there.
If i also can use this feature, i would also be very happy.

**Comment by jerome-diver** [comment]
@LordReg Maybe by define it through layout config variable.
if you add the region inside the layout definition (at 1 by default), then it should be possible to manage this because at jump time (when you change ws), it should be then able to consider on which region the layout is define and stack/manage the workspaces to be in front for each region only from there.
If i also can use this feature, i would also be very happy.

**Comment by darthzyklus** [comment]
@hexdsl did you solve this? I would like this feature too.

**Comment by vale981** [comment]
+1 I would like it as well :)

**Comment by vale981** [comment]
I have implemented something like that in #335.

**Comment by n3oney** [comment]
Any updates on this?
